velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -496,6 +496,12 @@ uint32_t HiveDataSink::appendWriter(const HiveWriterId& id) {
   options.memoryPool = connectorQueryCtx_->connectorMemoryPool();
   options.compressionKind = insertTableHandle_->compressionKind();
   options.setMemoryReclaimer = connectorQueryCtx_->setMemoryReclaimer();
+  options.kOrcWriterMaxStripeSize =
+      std::optional(HiveConfig::getKOrcWriterMaxStripeSize(
```

Comment:
@majetideepak can we consider to merge connector properties into config in connector query context? Thanks!

Reply:
@xiaoxmeng yes, we should. I will fix this.

Reply:
https://github.com/facebookincubator/velox/pull/6823